### ISSUE TITLE
Casting not working?

### ISSUE THREAD

**Issue by zeronullempty** [wako-app/wako#92]
Tried from both the android and iOS apps w/ a membership and I'm getting nothing showing up. Anyone else having this issue?

**Comment by jbtjr** [comment]
Yep same, real debrid have changed servers that’s the problem

**Comment by zeronullempty** [comment]
> Yep same, real debrid have changed servers that’s the problem

Nice to know I'm not the only one with an issue. Thanks. Hopefully it can be fixed but there have not been any commits in 7 months :(

**Comment by jbtjr** [comment]
Yep I’ve heard as well that they haven’t updated the server in years, can only hope for the best

**Comment by zeronullempty** [comment]
Hopefully @JumBay can take a look soon!

**Comment by JumBay** [comment]
Hi,
What do you mean in "casting not working"?
From the file explorer/resume feature you don't see the cast option? Or when your cast it, your android device tries to play but nothing happen? If the later then it's because the link is not playable by your chromecast device. Links are returned by add-ons. 

If you want to try the cast feature, inside wako go to settings, click 7+ times on the app version at the bottom, this will activate the developer mode (list bellow the app version). 
On this developer list, click on "open video" and try to cast the demo video. If it works it confirms that the links returned by add-ons are not playable.
